src/server.js

```diff
@@ -162,6 +162,11 @@ server.post(
     (req, res, next) => renderImage(req.body, res, next, tilePath)
 )
 
+server.get(
```

Comment:
Instead of adding a health check specific endpoint, what about adding a more generic root endpoint that can incidentally be used for health checks?

For example, something that lists the available routes registered on the server:

```
server.get({ url: '/' }, (req, res) => {
    const methods = ['GET', 'POST']
    const routes = {}
    methods.forEach((method) => {
        server.router.routes[method].forEach(({ spec: { url } }) => {
            if (!routes[url]) {
                routes[url] = []
            }
            routes[url].push(method)
        })
    })
    res.send({
        routes
    })
})
```